conans/test/functional/command/search_test.py

```diff
@@ -1255,7 +1255,22 @@ def search_test(self):
 
 @unittest.skipIf(get_env("TESTING_REVISIONS_ENABLED", False), "No sense with revs")
 class SearchOutdatedTest(unittest.TestCase):
-    def search_outdated_test(self):
+
+    def test_search_outdated(self):
+
+        def validate_outdated_table(expected):
```

Comment:
This seems a bit interleaved with other checks, and a bit fragile for changes. Try to simplify the checks, or do a separate, very simple test.

Reply:
Done. I separated the test, also I using single package id, so it should not change in the future.